schoolbot/cogs/school/meal.py

```diff
@@ -1,16 +1,26 @@
 import asyncio
+import os
 
 import neispy
+import os
```

Comment:
why twice import os?

Reply:
Outdated Please Check the updated commit